test/JDBC/input/BABEL-3326-vu-verify.mix

```diff
@@ -0,0 +1,527 @@
+INSERT INTO dbo.babel_3326_Employees ( EmployeeName, EmployeeAddress, MonthSalary )
+VALUES ( 'Temp Name5' , 'Address Name 5', 4000)
+GO
+
+UPDATE dbo.babel_3326_Employees
+SET MonthSalary = 3333
+WHERE EmployeeID = 2;
+GO
+
+
+-- Check support for following syntax
+-- { ENABLE | DISABLE } TRIGGER { [ schema_name . ] trigger_name [ ,...n ] | ALL }  
+-- ON { table_name } [ ; ]
+
+DISABLE TRIGGER [TR_ins_안녕하세요_babel_3326_Employees] ON babel_3326_Employees
+GO
+
+-- TR_ins_안녕하세요_babel_3326_Employees should be disabled
+INSERT INTO dbo.babel_3326_Employees ( EmployeeName, EmployeeAddress, MonthSalary )
+VALUES ( 'Temp Name5' , 'Address Name 5', 4000)
+GO
+
+ENABLE TRIGGER [TR_ins_안녕하세요_babel_3326_Employees] ON [babel_3326_Employees]
+GO
+
+-- TR_ins_안녕하세요_babel_3326_Employees should be enabled
+INSERT INTO dbo.babel_3326_Employees ( EmployeeName, EmployeeAddress, MonthSalary )
+VALUES ( 'Temp Name5' , 'Address Name 5', 4000)
+GO
+
+DISABLE TRIGGER [dbo].[TR_ins_안녕하세요_babel_3326_Employees] ON [babel_3326_Employees]
+GO
+
+-- TR_ins_안녕하세요_babel_3326_Employees should be disabled
+INSERT INTO dbo.babel_3326_Employees ( EmployeeName, EmployeeAddress, MonthSalary )
+VALUES ( 'Temp Name5' , 'Address Name 5', 4000)
+GO
+
+ENABLE TRIGGER [dbo].[TR_ins_안녕하세요_babel_3326_Employees] ON babel_3326_Employees
+GO
+
+-- TR_ins_안녕하세요_babel_3326_Employees should be enabled
+INSERT INTO dbo.babel_3326_Employees ( EmployeeName, EmployeeAddress, MonthSalary )
+VALUES ( 'Temp Name5' , 'Address Name 5', 4000)
+GO
+
+
+DISABLE TRIGGER ALL ON babel_3326_Employees
+GO
+
+-- All triggers on table babel_3326_Employees should be disabled
+INSERT INTO dbo.babel_3326_Employees ( EmployeeName, EmployeeAddress, MonthSalary )
+VALUES ( 'Temp Name5' , 'Address Name 5', 4000)
+GO
+
+UPDATE dbo.babel_3326_Employees
+SET MonthSalary = 3333
+WHERE EmployeeID = 2;
+GO
+
+ENABLE TRIGGER ALL ON [babel_3326_Employees]
+GO
+
+-- All triggers on table babel_3326_Employees  should be enabled
+INSERT INTO dbo.babel_3326_Employees ( EmployeeName, EmployeeAddress, MonthSalary )
+VALUES ( 'Temp Name5' , 'Address Name 5', 4000)
+GO
+
+UPDATE dbo.babel_3326_Employees
+SET MonthSalary = 3333
+WHERE EmployeeID = 2;
+GO
+
+
+DISABLE TRIGGER [TR_ins_안녕하세요_babel_3326_Employees], [TR_upd_abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz_babel_3326_Employees] ON [babel_3326_Employees]
+GO
+
+-- TR_ins_안녕하세요_babel_3326_Employees and TR_upd_abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz_babel_3326_Employees should be disabled
+INSERT INTO dbo.babel_3326_Employees ( EmployeeName, EmployeeAddress, MonthSalary )
+VALUES ( 'Temp Name5' , 'Address Name 5', 4000)
+GO
+
+UPDATE dbo.babel_3326_Employees
+SET MonthSalary = 3333
+WHERE EmployeeID = 2;
+GO
+
+ENABLE TRIGGER [TR_ins_안녕하세요_babel_3326_Employees], TR_upd_abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz_babel_3326_Employees ON babel_3326_Employees
```

Comment:
For MS SQL, what's the result for : 

begin transaction
disable trigger
rollback 

Just wanted to make sure this DDL can be part of transaction

Reply:
Will that disable rollback by transaction in this case ?

Reply:
In MS SQL,  the result for:

```
begin transaction
disable trigger
rollback
```
is: the effect of disable trigger will also be rollbacked.